Contained/Contained/View Controllers/SettingsViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SettingsViewController.swift
+//  Contained
+//
+//  Created by Rebecca Overare on 5/18/20.
+//  Copyright © 2020 Rebecca Overare. All rights reserved.
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+    
+
+    @IBAction func toggleRoll(_ sender: UISwitch) {
+        
```

Comment:
Take the value of the switch and assign that value to an instance variable for our class object.